src/wallet/wallet.cpp

```diff
@@ -227,10 +227,10 @@ static std::set<std::string> g_loading_wallet_set GUARDED_BY(g_loading_wallet_mu
 static std::set<std::string> g_unloading_wallet_set GUARDED_BY(g_wallet_release_mutex);
 
 // Custom deleter for shared_ptr<CWallet>.
-static void ReleaseWallet(CWallet* wallet)
+static void FlushAndDelete(CWallet* wallet)
```

Comment:
In commit "wallet: rename ReleaseWallet to FlushAndDelete" (7a73b0b7d5e690235bcefca9b82f01442b37ad5e)

Not important, but maybe this should be called FlushAndDeleteWallet to be consistent with WaitForDeleteWallet.

Reply:
Sure. Done.